update-charts-dep.sh

```diff
@@ -73,6 +77,10 @@ echo "Updating quoting-service..."
 helm dep up ./quoting-service
 checkCommandResult
 
+echo "Updating transaction-request-service..."
```

Comment:
You need to update the publish.sh script with a similar update.

Reply:
added to package.sh